main.js

```diff
@@ -0,0 +1,578 @@
+const Keyboard = {
+  elements: {
+    main: null,
+    keysContainer: null,
+    textarea: null,
+    keys: [],
+  },
+  properties: {
+    keysArr: [
```

Comment:
you can create separate file and define keysArr there

Reply:
create separate file